app/controllers/comments_controller.rb

```diff
@@ -0,0 +1,25 @@
+class CommentsController < ApplicationController
+	def create
+		# byebug
+		@comment = current_user.comments.new(body: params[:comment][:body], commentable_type: params[:comment][:commentable_type], commentable_id: params[:comment][:commentable_id])
```

Comment:
Let's have a private method to filter these params first:
```ruby
def comment_params
  params.require(:comment).permit(:body, : commentable_type .....)
end
```
and the use this method here like this:
```ruby
@comment = current_user.comments.new(comment_params)
```